packages/js/src/session/session.ts

```diff
@@ -55,11 +56,19 @@ export class Session {
   }
 
   public async initialize(options?: InitializeSessionArgs): Promise<void> {
+    if (this.#options.subscriber?.subscriberId === options?.subscriber?.subscriberId) {
+      return;
+    }
+
     try {
       if (options) {
         this.#options = options;
       }
       const { subscriber, subscriberHash, applicationIdentifier } = this.#options;
+      let currentTimezone;
+      if (isBrowser()) {
+        currentTimezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+      }
```

Comment:
get the current timezone

Reply:
1. Will this update the subscriber's timezone data? 
2. If the answer for 1 is yes, will it upsert the timezone on every Inbox init??

Reply:
@djabarovgeorge yes, that's exactly what we wanted, it will be updated on every session init if not passed on the subscriber object